### PR TITLE
Adding text to readme for extra info

### PR DESCRIPTION
**Pull request by jburgerhout**
Adding wildcard text and an extra helpfull link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 - Navigate to your Homestead directory, for me that's `~/Homestead`.
 - Install NFS for vagrant: `vagrant plugin install vagrant-winnfsd`
-- SSH into the VM: `vagrant ssh`
+- SSH into the VM: `vagrant ssh %ip/hostname of the VM%`
```
Comment (patatjenl) [inline README.md]: This is optional, could you reflect this in the readme?

File: README.md [hunk 2]
```diff
@@ -25,3 +25,4 @@ sudo apt-get update && sudo apt-get install zsh -y && sh -c "$(curl -fsSL https:
 - https://laracasts.com/discuss/channels/general-discussion/for-those-who-find-homesteadvagrantvirtualbox-slow-on-windows
 - https://websanova.com/blog/laravel/speeding-up-homestead-on-windows-using-nfs
 - https://laracasts.com/discuss/channels/servers/how-do-i-setup-2-homestead-boxes-one-with-php-56-and-one-with-php-7
+- https://enrise.com/2012/12/git-and-vagrant-in-a-windows-environment/
```
Comment (winfried-van-loon) [inline README.md:28]: Cool, thanks!

File: README.md (hunk 1 above)
Comment (winfried-van-loon) [inline README.md:6]: This is optional, could you reflect this in the readme?

**Review by winfried-van-loon — CHANGES_REQUESTED** [review]
Please fix the request :)

**Pull request merged**